theano/sandbox/cuda/blas.py

```diff
@@ -417,13 +419,14 @@ def __hash__(self):
         return hash(type(self)) ^ hash(self.inplace)
 
     def __setstate__(self, dct):
-        inplace = dct.get('inplace', True)
-        if inplace:
-            self.destroy_map = {0: [0]}
-        self.inplace = inplace
+        self.__dict__.update(dct)
 
-    def __getstate__(self):
-        return dict(inplace=self.inplace)
+        # Correctly reload older pickles where _op_use_c_code and
+        # destroy_map were not saved
```

Comment:
I removed the `__getstate__` method, so we use the default one.
GitHub may display the diff in a confusing way, but that code is in `__setstate__`.